### PR TITLE
Perception: Ransac lane fitting

### PR DESCRIPTION
**Pull request by xiaoshuliu**
1. Now using Ransac fitting instead of least square fitting. Process time has been tested.
2. Modified logic for using center lane. For close loop test purpose.

### CODE REVIEW

File: modules/perception/obstacle/camera/lane_post_process/common/util.h [hunk 1]
```diff
@@ -164,6 +164,81 @@ bool IterativeFitting(std::vector<Eigen::Matrix<T, 2, 1>> *pos_vec,
   return true;
 }
 
+template <typename T = ScalarType>
+bool RansacFitting(std::vector<Eigen::Matrix<T, 2, 1>> *pos_vec,
+             Eigen::Matrix<T, MAX_POLY_ORDER + 1, 1> *coeff,
+             const int max_iters = 100,
```
Comment (lianglia-apollo) [inline modules/perception/obstacle/camera/lane_post_process/common/util.h]: Please use input params first, then output params.

File: modules/perception/obstacle/camera/lane_post_process/common/util.h [hunk 2]
```diff
@@ -164,6 +164,81 @@ bool IterativeFitting(std::vector<Eigen::Matrix<T, 2, 1>> *pos_vec,
...
+             const int N = 5, double inlier_thres = 0.1) {
+  if (coeff == NULL) {
```
Comment (lianglia-apollo) [inline modules/perception/obstacle/camera/lane_post_process/common/util.h]: use nullptr

File: modules/perception/obstacle/camera/lane_post_process/common/util.h (hunk 1 above)
Comment (xiaoshuliu) [reply modules/perception/obstacle/camera/lane_post_process/common/util.h]: @lianglia-apollo cannot put parameter with default value before those without default value

File: modules/perception/obstacle/camera/lane_post_process/common/util.h (hunk 2 above)
Comment (xiaoshuliu) [reply modules/perception/obstacle/camera/lane_post_process/common/util.h]: @lianglia-apollo Changed. Thanks.

**Pull request merged**